#!/usr/bin/env python
PACKAGE = "baxter_perception"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("HueH",    double_t,    0, "Max value Hue channel", 0.5,  0.0, 1.0)
gen.add("HueL",    double_t,    0, "Min value Hue channel", 0.2,  0.0, 1.0)
gen.add("Sat",     double_t,    0, "Saturation value"     , 0.2,  0.0, 1.0)
gen.add("Val",     double_t,    0, "Value value"          , 0.2,  0.0, 1.0)
gen.add("DistMax", int_t,       0, "Maximum Distance"     ,  900,   0,3000)
gen.add("DistMin", int_t,       0, "Minimum Distance"     ,  600,   0,3000)

exit(gen.generate(PACKAGE, "baxter_perception", "baxter_perception"))


